core/server/master/src/main/java/alluxio/master/file/meta/Inode.java

```diff
@@ -262,6 +263,41 @@ public T replaceAcl(List<AclEntry> entries) {
     return setAcl(entries);
   }
 
+  /**
+   * Update Mask for the Inode.
```

Comment:
Comment on when this should be called. Can this just be a part of the update process?

Reply:
made it part of the update process.